config/initializers/data_source_adapters.rb

```diff
@@ -0,0 +1,7 @@
+require 'data_source_adapters'
+require 'data_source_adapters/base'
+require 'data_source_adapters/standard_adapter'
+
+require 'data_source_adapters/postgresql_adapter'
+require 'data_source_adapters/mysql2_adapter'
+require 'data_source_adapters/redshift_adapter'
```

Comment:
Is this config file required?
These files are located under app/models, so it seems to be autoloaded without this config.